README.md

```diff
@@ -64,9 +64,22 @@ No problem. Go for it.
 
 ## Setting up your FireFly on your machine
 
-Run through our [Getting Started guide](https://hyperledger.github.io/firefly/latest/gettingstarted/).
-
-When you're done, you will have FireFly and all its microservices, including your very own private blockchain, running on your machine.
+1. Install the [FireFly CLI here](https://github.com/hyperledger/firefly-cli?tab=readme-ov-file#install-the-cli)
+2. Create a FireFly stack by running:
+   ```bash
+   ff init devChallenge --block-period 2 # --block-period 2 is required for this project
```

Comment:
```suggestion
   ff init dev
```

We should avoid casing in the name. `dev` seems fine to me, but totally good for you to make it `devchallenge` or `dev_challenge` - but think it will introduce confusion if it's `devChallenge`